backend/src/prompts/templates/generate-knowledge-graph-model.j2

```diff
@@ -0,0 +1,102 @@
+You are an expert in Neo4j and you are an ESG specialist.
+
+Your task is to analyze the structure data and create intuitive Neo4j model, focusing on entities, attributes, and relationships using your ESG specialist skills.
```

Comment:
rephrased, thanks. I was going to leave the csv part out because when the file gets consumed by our app, the commas and new lines get taken out and the whole thing gets converted to python lists before being reconverted back into a string format to go into the prompt. Now that you say that I think I need to update promptfoo because promptfoo has it looking like a csv file when actually it probably looks totally different, I'll make a fix